src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -2461,6 +2482,37 @@ components:
           description: A unique reference for a FutureWorkInterests resource (if it already exists).
           example: c88a6c48-97e2-4c04-93b5-98619966447b
 
+    ### Conversations ###
+    ConversationType:
+      title: ConversationType
+      description: The categories of Conversation that a Prisoner may have recorded against their Learning and Work Progress.
+      type: string
+      enum:
+        - INDUCTION
+        - GENERAL
+        - REVIEW
+
+    CreateConversationRequest:
+      title: CreateConversationRequest
+      description: A request to create a Prisoner Conversation.
+      type: object
+      properties:
+        prisonId:
+          type: string
+          description: The identifier of the prison that the prisoner is currently resident at.
+          example: BXI
+        type:
+          $ref: '#/components/schemas/ConversationType'
+        note:
```

Comment:
For now `note` is a string and a required field, but it's possible in time this may become an optional string array.

It may be the case that at some point a prisoner "Conversation" could contain items other than notes, and a conversation may eventually contain many notes.